src/lib/modernExtend.ts

```diff
@@ -22,10 +22,11 @@ function getEndpointsWithInputCluster(device: Zh.Device, cluster: string | numbe
 }
 
 const timeLookup = {
-    '1_HOUR': 3600,
     'MAX': 65000,
+    '1_HOUR': 3600,
     '30_MINUTES': 1800,
     '10_SECONDS': 10,
+    'IMMEDIATE': 0,
```

Comment:
I propose to not add it here but instead add it to the function and map it to `0` there (since this is not really a time):

```
export function quirkPendingRequestTimeout(timeout: keyof typeof timeLookup | 'immediate'): ModernExtend {
```

Reply:
Yeah that why i tried to get it to also just take a number, I’ll revisit again and if that fails add the mapping in the function.

Reply:
Came up with a solution on my evening walk, should have gone earlier but it's been cold lately 🥶